src/icons/fileIcons.ts

```diff
@@ -2353,6 +2353,10 @@ export const fileIcons: FileIcons = {
         `.nanostagedrc`,
       ],
     },
+    {
+      name: 'docusaurus',
+      fileNames: ['docusaurus.config.js']
```

Comment:
![image](https://github.com/PKief/vscode-material-icon-theme/assets/81070564/7bab6f51-b5dc-47c4-af29-8366ef1de649)

```suggestion
      fileNames: ['docusaurus.config.js'],
```